src/library/dlgcoverartfullsize.cpp

```diff
@@ -46,6 +46,14 @@ DlgCoverArtFullSize::DlgCoverArtFullSize(QWidget* parent, BaseTrackPlayer* pPlay
     setupUi(this);
 }
 
+void DlgCoverArtFullSize::closeEvent(QCloseEvent* event) {
+    // Since the same instance if opened again by the same parent widget
```

Comment:
Do we need to handle the case if DlgCoverArtFullSize does not have a parent? In this case it should be closed.

Reply:
> ...we have to keep using the whole screen. Centering the window inside the `DlgTrackInfo` wouldn't make sense.

It centers now the window on the screen in which the preferences window is currently open. This is the expected behaviour in my opinion.

Reply:
Yes, that's what I mentioned. It works as intended. Only the description of this PR is slightly misleading.

Reply:
The wording "in the center of the mixxx window" wrong. "in the center of the screen where the parent window is displayed" would be correct.